### PR TITLE
crypto: Enable CC310 for all nrf91 series SoCs

### PR DESCRIPTION
**Pull request by joerchan**
Enable the cryptocell variant CC310 for all nrf91 series of SoCs.

### CODE REVIEW

**Review by frkv — APPROVED** [review]
LGTM

**Review by Vge0rge — APPROVED** [review]

**Review by Vge0rge — APPROVED** [review]

File: nrf_security/src/drivers/builtin/CMakeLists.txt [hunk 1]
```diff
@@ -6,73 +6,5 @@
...
-
-# Add builtin functionality (if enabled)
-if(DEFINED src_crypto_builtin)
-  target_sources(${mbedcrypto_target}
-    PRIVATE
-      src_crypto_builtin
-  )
-endif()
+# No additional sources needed
+# Sources have already been added in nrf_security/src/legacy/CMakeLists.txt
```
[the remark this answers is not in the document]
Comment (frkv) [reply nrf_security/src/drivers/builtin/CMakeLists.txt:10]: Empty files give less value. Please remove

File: nrf_security/src/drivers/nrf_oberon/CMakeLists.txt [hunk 2]
```diff
@@ -33,8 +33,16 @@ if (COMPILE_PSA_APIS)
     oberon_key_pair.c
     oberon_mac.c
     oberon_rsa.c
-    oberon_ctr_drbg.c
-    oberon_hmac_drbg.c
+  )
+
+  append_with_prefix_ifdef(PSA_CRYPTO_DRIVER_ALG_CTR_DRBG_OBERON
```
Comment (frkv) [inline nrf_security/src/drivers/nrf_oberon/CMakeLists.txt:38]: CONFIG_ prefixed is used earlier in this file. Please use this for consistency

File: nrf_security/src/drivers/nrf_oberon/CMakeLists.txt [hunk 3]
```diff
@@ -33,8 +33,16 @@ if (COMPILE_PSA_APIS)
...
+    src_crypto_oberon ${OBERON_PSA_CRYPTO_PATH}/oberon/drivers/
+      oberon_hmac_drbg.c
+  )
+
+  append_with_prefix_ifdef(PSA_CRYPTO_DRIVER_ALG_HMAC_DRBG_OBERON
```
Comment (frkv) [inline nrf_security/src/drivers/nrf_oberon/CMakeLists.txt:43]: CONFIG_ prefixed is used earllier in this file. Please use this for consistency

**Review by vili-nordic — APPROVED** [review]

File: nrf_security/src/drivers/builtin/CMakeLists.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (joerchan) [reply nrf_security/src/drivers/builtin/CMakeLists.txt:10]: This will be removed later anyway. Minimizing changes done in this PR now.

**Pull request merged**